### PR TITLE
4 implement loginactivity

### PR DESCRIPTION
**Pull request by bkaryna**

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -9,10 +9,16 @@
         android:roundIcon="@mipmap/ic_launcher_round"
         android:supportsRtl="true"
         android:theme="@style/Theme.PartyMaker">
-        <activity android:name=".activities.MainActivity">
+        <activity android:name=".activities.LoginActivity">
             <intent-filter>
                 <action android:name="android.intent.action.MAIN" />
+                <category android:name="android.intent.category.LAUNCHER" />
```
Comment (ixrk) [inline app/src/main/AndroidManifest.xml:15]: LoginActivity shouldn't be treated as `MAIN` nor `LAUNCHER`.

**Review by ixrk — CHANGES_REQUESTED** [review]
Please correct intent-filters. Now both activities are visible in Android app launcher.
Try to make `MainActivity` redirect users to the `LoginActivity` only if they're not logged in (for now it will be just `if (true)`)
UI is nice 😎

File: app/src/main/java/com/github/partymakers/partymaker/activities/LoginActivity.java [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_login);
+    }
+
+    public void loginSuccessful(View view) {
+        Intent intent = new Intent(this, MainActivity.class);
+        startActivity(intent);
```
Comment (ixrk) [inline app/src/main/java/com/github/partymakers/partymaker/activities/LoginActivity.java:20]: Use `finish` method instead.

**Review by ixrk — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/github/partymakers/partymaker/activities/LoginActivity.java (hunk 2 above)
Comment (ixrk) [reply app/src/main/java/com/github/partymakers/partymaker/activities/LoginActivity.java:20]: Also, if it's possible, make `loginSuccessfull()` private.

Comment (bkaryna) [reply app/src/main/java/com/github/partymakers/partymaker/activities/LoginActivity.java:20]: I'm afraid it's not possible. The app crashes in both cases. 
image

Comment (ixrk) [reply app/src/main/java/com/github/partymakers/partymaker/activities/LoginActivity.java:20]: leave it public then

**Review by ixrk — APPROVED** [review]
🚀

**Pull request merged**